recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
 
 build:
   number: 0
-  skip: True  # [py3k]
+  skip: True  # [osx and py3k]
```

Comment:
Sure. I thought I should just let you know why you did not get Python 3.6 there. But now that you know how to rerender you can add that whenever vtk is ready.